pkg/pillar/types/volumetypes.go

```diff
@@ -132,6 +137,12 @@ type VolumeStatus struct {
 	Target                  zconfig.Target
 	CustomMeta              string
 
```

Comment:
@zedi-pramodh , many files of types are being changed by this PR... will any of these changes impact other components that use pillar types? For instance, edgeview, installer, wwan, vtpm, newlog.... If so, you should keep it in sync... although I think is better to update those in a dedicated PR....

Reply:
I do see all these types files are already not in sync between pillar and other components, even before this commit.
I will probably do a new PR to sync those.